### PR TITLE
[WOKE] Removes species pay modifiers

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

I've been thinking about it and I don't think this is conclusive to a fun game.

# Why is this good for the game?
This was originally a conditional for letting non-humans be certain heads. It was funny haha cartoon racism in the moment, but I don't think it's conclusive to good game balance or culture design to promote arbitrary "you're a weird furry so you're paid less", even if it doesn't matter due to economy being unbalanced.

# Wiki Documentation

Paycheck Modifier needs to be removed from every species

# Changelog

:cl:  
rscdel: Removes species pay modifiers
/:cl:

### CODE REVIEW

**Comment by cowbot92** [comment]
we should make coders get paid 5x more

**Review by JamieD1 — APPROVED** [review]
Looks good to me

**Comment by YashinHak** [comment]
i dont care about money but i also dont like lizards, luckily one of the mords has found us a solution: remove it for every species except lizards.

**Comment by AMyriad** [comment]
Lore team good with this?

**Comment by DeadlyBag** [comment]
if you care about this PR (in a negative sense or a positive one) you are a god damn lizard sympathizer.

Honestly though, this is probably one of the worst ways to push "IC racism"
I do think that such a topic can be quite interesting when explored and can give tension but this... just doesnt do much?
Im sure there are other ways to give players the option to explore IC racism, a good way for example is that humans take priority as acting captains, it effectively creates a tension between head A and head B while using racism. 
All in all, even if you want IC racism explored, the pay mods do a terrible job at it and at best are just there to prevent non-humans from buying vendor shit or for side remarks of "you get paid less dumbass lol"


Tl;dr Racism is an interesting topic to explore but paymods do a terrible job at it which is why i support this PR

**Review by Moltijoe — APPROVED** [review]

**Comment by solwashere** [comment]
> Lore team good with this?

Biome is above lore team, but sure
